frontend/javascripts/admin/time/time_line_view.js

```diff
@@ -300,6 +300,7 @@ class TimeLineView extends React.PureComponent<Props, State> {
               </FormItem>
               <FormItem {...formItemLayout} label="Date">
                 <RangePicker
+                  showTime
```

Comment:
I would propose to disable the minutes and seconds if that makes the ui less complicated. According to the antd docs you can supply an object to `showTime` which will be passed on to the TimePicker component (https://ant.design/components/time-picker/#API). Does that work for our antd version and if so does it make the UI simpler?

Reply:
Sadly the less complicated version seems not to be available in our antd version. We would need to upgrade antd for this. On the newest node version the `RangePicker` could look like [this](https://codesandbox.io/s/oz6iv).

I choose to keep the option to select minutes and removed only the seconds as the selection of minutes gives the possibility for detailed insight into the tracked time of users. 
Additionally only having hours looks pretty awful in our antd version